### PR TITLE
chore: Remove max_length from WebUI

### PR DESCRIPTION
**Pull request by gt2345**
## Ticket

[MD-527]

## Description

Part of searcher context removal project.
Remove `max_length` for searchers in WebUI

## Test Plan

Continue/Fork/HPSearch work as expected with the right backend.
Tests passed.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[MD-527]: https://hpe-aiatscale.atlassian.net/browse/MD-527?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/components/ExperimentContinueModal.tsx [hunk 1]
```diff
@@ -125,9 +116,7 @@ const ExperimentContinueModalComponent = ({
     : [ExperimentCopyMapping, experimentEntityCopyMap];
   const actionCopy = actionCopyMap[modalState.type];
 
-  useEffect(() => setOriginalConfig(experiment.configRaw), [experiment]);
-
-  const requiredFields = useMemo(() => [EXPERIMENT_NAME, MAX_LENGTH], []);
+  const requiredFields = useMemo(() => [EXPERIMENT_NAME], []);
```
Comment (johnkim-det) [inline webui/react/src/components/ExperimentContinueModal.tsx:119]: nit: do we need `requiredFields`?

File: webui/react/src/components/ExperimentCreateModal.tsx [hunk 2]
```diff
@@ -120,7 +117,7 @@ const ExperimentCreateModalComponent = ({
     ? `Fork ${capitalize(entityCopy.experiment)} ${experiment.id}`
     : `Continue ${capitalize(entityCopy.trial)} ${trial?.id}`;
 
-  const requiredFields = useMemo(() => [EXPERIMENT_NAME, MAX_LENGTH], []);
+  const requiredFields = useMemo(() => [EXPERIMENT_NAME], []);
```
Comment (johnkim-det) [inline webui/react/src/components/ExperimentCreateModal.tsx:120]: ditto

File: webui/react/src/components/ExperimentCreateModal.tsx [hunk 3]
```diff
@@ -25,8 +25,6 @@ import handleError, {
 import {
```
Comment (johnkim-det) [inline webui/react/src/components/ExperimentCreateModal.tsx:25]: nit: can we use `capitalizeWord` on the submit button text in this modal?

**Review by johnkim-det — APPROVED** [review]
noted some nits but web changes LGTM

**Review by azhou-determined — APPROVED** [review]
lgtm

File: webui/react/src/components/ExperimentContinueModal.tsx (hunk 1 above)
Comment (gt2345) [reply webui/react/src/components/ExperimentContinueModal.tsx:119]: I think it's no harm keeping it, in case we want to expand it in the future

Comment (ashtonG) [reply webui/react/src/components/ExperimentContinueModal.tsx:119]: consider pulling this and the other array out of the component, given that they are static values.

**Pull request merged**